www/base/src/app/schedulers/schedulers.tpl.jade

```diff
@@ -3,16 +3,22 @@
       uib-tab(heading="All schedulers")
         table.table.table-hover.table-striped.table-condensed
           tr
+              td Enabled
               td Scheduler Name
               td Master
           tr(ng-repeat='scheduler in schedulers | orderBy: ["name"]')
+              td
+                input(type="checkbox", ng-model="scheduler.enabled", ng-change="change()")
```

Comment:
ng-change="change(scheduler)"

so that you get the schedulerid and the new value